### DISCUSSION TITLE
Einspeiselimit via MQTT mit Homeassistant Automatisation ändern

### DISCUSSION THREAD

**Discussion by daToni56** [tbnobody/OpenDTU#1937] [Q&A]
Mit der Automatisation versende ich an das Topic "ODoB/114184621287/status/limit_relative" mit Payload "40" das Relative Limit 40%.
Es ist mit dem MQTT Explorer auch sichtbar, wird aber kurz danach wieder von OpenDtu mit dem in OpenDTU aktuell eingestellten Wert überschrieben.
Mit dem Schieberegler kann ich den Wert aber problemlos verstellen
Was mache ich hier falsch?
Danke
Anton
![grafik](https://github.com/tbnobody/OpenDTU/assets/126894707/99f8b150-a091-43b3-ac84-2e42f225d51f)
![grafik](https://github.com/tbnobody/OpenDTU/assets/126894707/2705a1a8-c130-4656-a12d-20944efa49ad)

**Comment by tbnobody** [comment]
Du darfst nicht die Status topics verwenden sondern die cmd topics
 Siehe https://www.opendtu.solar/firmware/mqtt_topics/#inverter-limit-specific-topics

**Reply by daToni56** [reply]
Danke für die schnelle Info, werde es am Abend umsetzen
Gruß
Anton
